lib/sidekiq/killswitch.rb

```diff
@@ -45,7 +44,8 @@ def blackhole_remove_worker(worker_name)
 
     def blackhole_worker?(worker_name)
       redis_pool do |redis|
-        redis.hexists(BLACKHOLE_WORKERS_KEY_NAME, class_to_str(worker_name))
+        hexists = redis.hexists(BLACKHOLE_WORKERS_KEY_NAME, class_to_str(worker_name))
+        hexists == 1 || hexists == true
```

Comment:
Can we pull this check into like an `exists` general helper method? Seems more foolproof than having to write out this check every time (and opposite one for `false`)